### PR TITLE
completed files modification

### PR DESCRIPTION
**Pull request by odyred**

### CODE REVIEW

File: tests/Class1.cs [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+{
+    public class SolveProblem
+    {
+        public void Solve()
+        {
+            const int iterations = 1000;
+            const double youngModulus = 2.1e8;
+
+            var domainMapper = new CantileverStochasticDomainMapper(new [] { 0d, 0d, 0d });
+            var realizer = new GiannisStructuralStochasticRealizer(youngModulus, domainMapper);
```
Comment (dimtsap) [inline tests/Class1.cs]: Rename the **GiannisStructuralStochasticRealizer** and **Class1** with a more suitable names.

File: tests/Class1.cs [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+            var evaluator = new StructuralStochasticEvaluator(youngModulus, domainMapper);
+            var m = new MonteCarlo(iterations, realizer, evaluator);
+            m.Evaluate();
+        }
```
Comment (dimtsap) [inline tests/Class1.cs]: The test has no assertion. This will automatically fail.

**Review by dimtsap — CHANGES_REQUESTED** [review]

File: tests/Class1.cs (hunk 1 above)
Comment (odyred) [reply tests/Class1.cs]: done

File: tests/Class1.cs (hunk 2 above)
Comment (odyred) [reply tests/Class1.cs]: done

**Review by dimtsap — APPROVED** [review]

**Pull request merged**